### PR TITLE
Fixed '$templateNames' variable name and type.

### PR DESCRIPTION
**Pull request by GerbrandStap**
When trying to provision a new Dynamics instance, I noticed that the template check boxes were not working.
I think my change should fix the problem, but I haven't got a validated organization (yet) to add an extension to, so I was not able to test the fix.
The problems I tried to fix:
- The name of the variable that was initialized was different from the variable passed to the ProvisionOnlineInstance.ps1 script.
- The initialized variable was not a string array (but just a string).

### CODE REVIEW

**Comment by WaelHamze** [comment]
@GerbrandStap Thx a lot for submitting this PR.

What is the actual issue/re-pro that you are facing so I am able to test this?

**Comment by GerbrandStap** [comment]
I wanted to provision an instance with the Field Service template. But it didn't matter if the Field Service check box in the Provision Instance task was checked or not. In each case the only installed solution (as seen in the admin page) was the 'crm hub' and 'odata' solution. Checking all templates in the task didn't do anything either.
I was able to provision a new instance with Field Service from my own machine using the appropriate Powershell command, so it is possible with the license I used.

So, in short: the template check boxes in the provision instance task don't do anything.

**Comment by WaelHamze** [comment]
@GerbrandStap thanks for the this, I'll merge it and it will be available in the next release
